plugins/ltac/g_auto.mlg

```diff
@@ -155,6 +155,7 @@ TACTIC EXTEND unify
       let state = Hints.Hint_db.transparent_state t in
       Tactics.unify ~state x y
   }
+| ["convert" constr(x) constr(y) ] -> { Tactics.convert x y }
```

Comment:
If you add in its own TACTIC EXTEND macro this will be handled as a normal Ltac identifier, which is better for parsing purposes.